math/binary_exponent.cpp

```diff
@@ -19,53 +20,116 @@
  * \f}
  * Hence to calculate 2^10 we only need to multiply \f$2^8\f$ and \f$2^2\f$
  * skipping \f$2^1\f$ and \f$2^4\f$.
+ *
+ * @author [Mann Mehta](https://github.com/mann2108)
```

Comment:
You can also add yourself as you've helped update this algorithm to our standards. 🙂
```suggestion
 * @author [Mann Mehta](https://github.com/mann2108)
 * @author [ewd00010](https://github.com/ewd00010)
```